### PR TITLE
css-canvas-medical | Nick Hernandez

### PR DESCRIPTION
**Pull request by Nickolas-Hernandez**
Screen Shot 2021-01-23 at 4 59 29 PM
Screen Shot 2021-01-23 at 4 59 50 PM
Screen Shot 2021-01-23 at 4 59 59 PM

### CODE REVIEW

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+.learn-more-row {
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* Footer links and location info section */
+.links-and-location-section {
+  background-color: #525351;
+  color: white;
+  height: 40vh;
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:172]: This is going to seem random, but 99.999% _never_ use vh for hardly anything. Favor `px` for height if it works.

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,202 @@
+/* Globals and Layouts */
+html {
+  font-family: "Avenir", sans-serif;
+}
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:4]: Typically we put this on `body` and not `html`, since `body` is the only thing that is visible.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Fantasic, just a couple minor tweak.

**Comment by Nickolas-Hernandez** [comment]
Screen Shot 2021-01-28 at 2 47 16 PM


Added font-family property to body selector, and changed all vh values to px.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**